src/main/kotlin/com/kotlin/boardproject/global/service/CustomOAuth2UserService.kt

```diff
@@ -1,19 +1,19 @@
-package com.kotlin.boardproject.service
+package com.kotlin.boardproject.global.service
 
-import com.kotlin.boardproject.auth.OAuth2UserInfo
-import com.kotlin.boardproject.auth.OAuth2UserInfoFactory
-import com.kotlin.boardproject.auth.ProviderType
-import com.kotlin.boardproject.auth.UserPrincipal
-import com.kotlin.boardproject.common.enums.ErrorCode
-import com.kotlin.boardproject.common.exception.InternalServiceException
-import com.kotlin.boardproject.common.exception.OAuthProviderMissMatchException
-import com.kotlin.boardproject.model.User
-import com.kotlin.boardproject.repository.UserRepository
+import com.kotlin.boardproject.domain.user.domain.User
```

Comment:
🚫 **[ktlint]** <standard:import-ordering> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Imports must be ordered in lexicographic order without any empty lines in-between with "java", "javax", "kotlin" and aliases in the end